multimedia/scrcpy/Portfile

```diff
@@ -4,8 +4,9 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           meson 1.0
 PortGroup           compiler_blacklist_versions 1.0
+PortGroup           legacysupport 1.1
```

Comment:
additionally, you should add
```legacysupport.newest_darwin_requires_legacy ??``` to specify which macOS versions require it and please add a comment near that line explaining for which missing functions it is added

Reply:
Addressed in https://github.com/macports/macports-ports/pull/20727/commits/fccab39fc1ba10b17275c8208f29723316493073.